cmd/echo.lisp

```diff
@@ -11,4 +11,4 @@
 
 ;initialize pipe details and command args, abort on error
 (when (and (defq slave (create-slave)) (defq args (options slave usage)))
-	(each print (slice 1 -1 args)))
+	(print (apply cat (map (lambda (_) (cat _ " ")) (slice 1 -1 args)))))
```

Comment:
Won't this have an extra space added on the line that goes to stdout ? ;)

Reply:
LOL. Oops.